porch/apiserver/pkg/registry/porch/functions.go

```diff
@@ -16,13 +16,13 @@ package porch
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"strings"
 
 	"github.com/GoogleContainerTools/kpt/porch/api/porch/v1alpha1"
 	configapi "github.com/GoogleContainerTools/kpt/porch/api/porchconfig/v1alpha1"
 	"github.com/GoogleContainerTools/kpt/porch/engine/pkg/engine"
+	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Nit: in OSS we tend to import this as `apierrors`, just to avoid the conflict with errors in stdlib

Reply:
renamed. thank you!